src/functionalTest/resources/application.conf

```diff
@@ -4,3 +4,10 @@ storage-account-url=${STORAGE_ACCOUNT_URL}
 signing-key-der-base64=${SIGNING_KEY_DER_BASE64}
 proxy-host=proxyout.reform.hmcts.net
 proxy-port=8080
+idam-api-url=https://idam-api.aat.platform.hmcts.net
+s2s-url=http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
+core-case-data-api-url=http://ccd-data-store-api-aat.service.core-compute-aat.internal
```

Comment:
best to keep it consistent with other config so we can use env var here and define it in Jenkins file

Reply:
Done